src/main/java/mate/academy/Main.java

```diff
@@ -2,16 +2,29 @@
 
 import java.time.LocalDate;
 import java.time.LocalDateTime;
+import mate.academy.exception.AuthenticationException;
+import mate.academy.exception.RegistrationException;
+import mate.academy.lib.Injector;
 import mate.academy.model.CinemaHall;
 import mate.academy.model.Movie;
 import mate.academy.model.MovieSession;
+import mate.academy.model.User;
+import mate.academy.security.AuthenticationService;
 import mate.academy.service.CinemaHallService;
 import mate.academy.service.MovieService;
 import mate.academy.service.MovieSessionService;
 
 public class Main {
-    public static void main(String[] args) {
-        MovieService movieService = null;
+    private static final Injector INJECTOR = Injector.getInstance("mate.academy");
+    private static final String BOB_EMAIL = "bob@email";
+    private static final String ALICE_EMAIL = "alice@email";
```

Comment:
The email addresses for `BOB_EMAIL` and `ALICE_EMAIL` are missing a domain suffix (e.g., '.com'). This may cause issues during registration and authentication as they do not conform to standard email formats.